gravitee-apim-e2e/api-test/use-case-test/src/management/publisher/endpoints/add-edit-remove-several-endpoints-to-group-and-use-them.spec.ts

```diff
@@ -13,8 +13,127 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { describe } from '@jest/globals';
+import { APIsApi } from '@management-apis/APIsApi';
+import { forManagementAsApiUser } from '@client-conf/*';
+import { afterAll, beforeAll, describe, expect } from '@jest/globals';
+import { succeed } from '@lib/jest-utils';
+import { ApisFaker } from '@management-fakers/ApisFaker';
+import { ApiEntity } from '@management-models/ApiEntity';
+import { PlansFaker } from '@management-fakers/PlansFaker';
+import { LifecycleAction } from '@management-models/LifecycleAction';
+import { PlanStatus } from '@management-models/PlanStatus';
```

Comment:
Hi @ctschacher, I just merge https://github.com/gravitee-io/gravitee-api-management/pull/2331.
Could you update imports with the new module names ? (like here https://github.com/gravitee-io/gravitee-api-management/pull/2331/files)

Reply:
ok